hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/utils/ActionsHelper.java

```diff
@@ -30,13 +30,14 @@ public class ActionsHelper {
      * @param frame the frame to represent
      * @return the {@link ContractAction} representing the frame as a call to a missing address
      */
-    public ContractAction createSynthActionForMissingAddressIn(@NonNull final MessageFrame frame) {
+    public ContractAction createSynthActionForMissingAddressIn(
+            @NonNull final MessageFrame frame, @NonNull final InvalidAddressContext invalidAddressContext) {
```

Comment:
why not just pass in the address?  InvalidAddressContext is not used for anything else AFAICT